cms/envs/common.py

```diff
@@ -917,6 +917,12 @@
 SESSION_SERIALIZER = 'openedx.core.lib.session_serializers.PickleSerializer'
 SESSION_COOKIE_DOMAIN = ""
 SESSION_COOKIE_NAME = 'sessionid'
+ADDITIONAL_NODE_ENV_VARS = {
+    'ADSK_PLUGIN_LINKTYPES': 'Download,Offer',
+    'ADSK_PLUGIN_FILETYPES': 'PDF,zip,Video,Design',
+    'ADSK_PLUGIN_ORIENTATIONS': 'Vertical,Horizontal',
+    'ADSK_PLUGIN_STYLES': 'Primary,Normal,Secondary'
+}
```

Comment:
We should put these into `lms.yml` and `studio.yml` (probably via the EDXAPP_ENV_EXTRA ansible vars), not into a codebase fork.